From 9d0c3158bc73292ca755477f60cbf5b5fd622fde Mon Sep 17 00:00:00 2001
From: Damien Douxchamps <ddsf@douxchamps.net>
Date: Thu, 12 Jan 2017 16:47:27 +0900
Subject: [PATCH] Fix bad variable type uint -> int

Type was change to uint in recent commit but it should have been INT.

Signed-off-by: Damien Douxchamps <ddsf@douxchamps.net>
---
 libdc1394/AUTHORS        | 1 +
 libdc1394/dc1394/bayer.c | 3 ++-
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/libdc1394/dc1394/bayer.c b/libdc1394/dc1394/bayer.c
index 10a4c1b..cbdc417 100644
--- a/dc1394/bayer.c
+++ b/dc1394/bayer.c
@@ -24,6 +24,7 @@
 #include <limits.h>
 #include <math.h>
 #include <stdlib.h>
+#include <stdint.h>
 #include <string.h>
 #include "conversions.h"
 
@@ -775,7 +776,7 @@ dc1394_bayer_Downsample(const uint8_t *restrict bayer, uint8_t *restrict rgb, in
 {
 	uint8_t *outR, *outG, *outB;
 	register int i, j;
-	uint tmp;
+	int tmp;
 	int st=sx*sy;
 	int p;
 	int sx2=sx<<1;
-- 
2.11.0

